independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/StorkClientRequestFilter.java

```diff
@@ -0,0 +1,68 @@
+package org.jboss.resteasy.reactive.client.impl;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.stork.Stork;
+import io.smallrye.stork.api.ServiceInstance;
+import java.net.URI;
+import java.net.URISyntaxException;
+import javax.annotation.Priority;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.ext.Provider;
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.client.spi.ResteasyReactiveClientRequestContext;
+import org.jboss.resteasy.reactive.client.spi.ResteasyReactiveClientRequestFilter;
+
+@Priority(100)
```

Comment:
This would need to be carefully documented (how to inject a filter executed before and after).

Reply:
Also, why the number 100?

Reply:
I think that we thought that this filter has to be applied before authentication and put 100 to be sure of that.

Reply:
If that's the case, it's best to reference `javax.ws.rs.Priorities.AUTHENTICATION`